plugins/org/README-alpha.md

```diff
@@ -0,0 +1,462 @@
+       - entity-card:org/user-profile
+   ```
+
+3. Then start the app, navigate to an entity's page and see the cards and contents in there.
+
+## Packages
+
+The `org` plugin can be automatically discovered, and it is also possible to enable it only in certain [environments](https://backstage.io/docs/conf/writing/#configuration-files). See [this](https://backstage.io/docs/frontend-system/architecture/app/#feature-discovery) packages documentation for more details.
+
+## Routes
+
+The `org` plugin exposes a external route that can be used to configure route bindings.
+
+| Key            | Type           | Description                        |
+| -------------- | -------------- | ---------------------------------- |
+| `catalogIndex` | External route | A route ref to Catalog Index page. |
+
+As an example, here is an association between the external catalog index page and a regular route from another plugin:
+
+```yaml
+# app-config.yaml
+app:
+  routes:
+    bindings:
+      # example binding org and catalog index pages
+      org.catalogIndex: catalog.catalogIndex
+```
+
+Route binding is also possible through code. For more information, see [this](https://backstage.io/docs/frontend-system/architecture/routes#binding-external-route-references) documentation.
+
+## Extensions
+
+### Entity Cards
+
+The `org` plugin provide some entity cards you can enable to customize the Software Catalog entity page.
+
+> [!IMPORTANT]
+> The order in which cards are listed in the configuration file will determine the order in which they appear in overview cards and tab lists on entity pages.
+
+See a complete cards list below:
+
+#### Entity Group Profile Card
+
+This [entity card](https://github.com/backstage/backstage/blob/master/plugins/catalog-react/api-report-alpha.md) extension allows you to view, edit, or update groups metadata, such as team avatar, name, email, parent, and child groups.
+
+| Kind          | Namespace | Name            | Id                              | Example                                                                                   |
+| ------------- | --------- | --------------- | ------------------------------- | ----------------------------------------------------------------------------------------- |
+| `entity-card` | `org`     | `group-profile` | `entity-card:org/group-profile` | <img alt="Entity Group Profile Card" src="./OrgGroupProfileEntityCard.png" width="200" /> |
+
+##### Disable
+
+This card is disabled by default when you install the `org` plugin, but to ensure the card will always be disabled or enabled regardless of the extension's default definition, add the following configuration:
+
+```yaml
+# app-config.yaml
+# example disabling the org group profile entity card extension
+app:
+  extensions:
+    # this is the extension id and it follows the naming pattern bellow:
+    # <extension-kind>/<plugin-namespace>:<extension-name>
+    # use false as value for disabling the extension and true for enabling
```

Comment:
we generally recommend just the id string:

```yaml
- entity-card:org/group-profile
```

But that makes me think, isn't this entire Disable section kind of not needed for cards? They are already disabled and I am pretty sure that the default definition (mentioned above) will never change. Isn't it the case then, that this entire section only leads to confusion? The installation section already showed how to enable them, that's "the installation flow" so to speak for these in the first place.

Reply:
I thought it was a similar risk to installing the latest version of a dependency, it's easy to remove the default disabled by accident and people start seeing a card they don't want... But I also agree that we generally install the plugin because we want to use all cards, perhaps there is no case where someone always wants to disable a specific card... Would you like me to remove it 🙂 ?

Reply:
i think so - very personal opinion though - the regular generic knowledge of how to enable and disable things should be enough (maybe pointing to a generic section) for ANY extension, without having to copy-paste the same block of info many many times all over 🤔

Reply:
Done ✅